recipes/anuga/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "anuga" %}
+{% set version = "3.1.8" %}
+{% set sha = "1f15690caf02ed9c45d088536bee22d10a41d1954077e884b3cdcc361d422904" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anuga-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<3.6]
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - m2w64-gcc                  # [win]
+    - libpython                  # [win]
+
+  host:
+    - python 
+    - libpython  # [win]
+    - pip
+    - cython
+    - numpy
+
+  run:
+    - python
+    - _openmp_mutex   # [linux]
+    - dill
+    - future
+    - gdal
+    - gitpython
+    - matplotlib-base
+    - meshpy
+    - netcdf4
+    - numpy
```

```suggestion
    - {{ pin_compatible('numpy') }}
```
The numpy version at compile time should match the numpy version at runtime.

---

Just want to double check: Does your compiled code link against the numpy C API?